src/components/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -20,7 +20,7 @@ export class ErrorBoundary extends React.Component<Props, State> {
         error: undefined,
     };
 
-    componentDidUpdate() {
+    componentDidCatch() {
```

Comment:
`componentDidCatch` definitely invoked in case of error (unlike `componentDidUpdate`)